hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/AbstractKeyDeletingService.java

```diff
@@ -244,4 +422,35 @@ public ScmBlockLocationProtocol getScmClient() {
   public AtomicLong getRunCount() {
     return runCount;
   }
```

Comment:
Can we just return `Long` instead of having every single caller call `get()` individually?

```suggestion
  public Long getRunCount() {
    return runCount.get();
  }
```